projects/noloan2/app/.gitignore

```diff
@@ -1 +1,3 @@
 /build
+/lib
+*.jar
```

Comment:
/lib/*.jar - ?

Reply:
That's good. All jar files, all files under lib.. :)